src/main/java/pages/HeaderMenu.java

```diff
@@ -0,0 +1,37 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import utils.WaitActions;
+
+public class HeaderMenu {
```

Comment:
Still you have your page fragments in the same package with your application pages. It's better to place them in a separate package. In other words, let's create new package, e.g., "fragments" or whatever, and then move HeaderMenu and so on there.